openmdao/utils/units.py

```diff
@@ -845,6 +845,11 @@ def _update_library(cfg):
 _UNIT_CACHE = {}
 
 
+def _is_unitless(units):
+    unit_meta = _find_unit(units) if units else None
+    return not unit_meta.is_dimensionless()
```

Comment:
Shouldn't this be `return unit_meta.is_dimensionless()` instead of `return not unit_meta.is_dimensionless()`?  Also,  you're not handling the case where unit_meta ends up being None.

Reply:
I removed the chance of it being None since it's private and protected by a none check in each instance. Fixed the `not` issue.